docs/content_management/field_types/field_type_reference/checkboxfield.md

```diff
@@ -46,4 +46,4 @@ $checkboxValue = new Checkbox\Value( true );
 
 ##### String representation
 
-As this Field Type is not a string but a boolean, it will return "1" (true) or "0" (false) in cases where it is cast to string.
+As this Field Type is not a string but a boolean, it will return "1" (true) or "0" (false) in cases where it is cast to string, and it is never considered as empty.
```

Comment:
Shouldn't it be
```suggestion
As this Field Type is not a string but a boolean, it will return "1" (true) or "0" (false) in cases where it is cast to string, and it is never considered empty.
```
?